dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ requests==2.5.1
 responses==0.3.0
 wheel
 twine
+Django==1.8.5
```

Comment:
ya, tox will be a good idea. i've seen that in some other projects. 

i agree. then let's target this task to add support for 1.8x only. we'll add more versions support in next phase.